angular.json

```diff
@@ -128,7 +130,7 @@
         "serve": {
           "builder": "@angular-builders/custom-webpack:dev-server",
           "options": {
-            "browserTarget": "cookbook:build"
```

Comment:
For some reason this is appearing in the diff - but it is already present on target branch